src/Uno.UWP/Storage/Pickers/FileOpenPicker.iOS.cs

```diff
@@ -75,6 +61,52 @@ private async Task<FilePickerSelectedFilesArray> PickFilesAsync(bool multiple, C
 			return new FilePickerSelectedFilesArray(files);
 		}
 
+		private UIViewController GetViewController(bool multiple, TaskCompletionSource<NSUrl?[]> completionSource)
```

Comment:
Can you move this to the top at its original place so that diff is simpler?

Reply:
done !